plugins/available/battery.plugin.bash

```diff
@@ -2,15 +2,23 @@ cite about-plugin
 about-plugin 'display info about your battery charge level'
 
 ac_adapter_connected(){
-    if command_exists acpi;
-    then
-        acpi -a | grep "on-line"
-        if [[ "$?" -eq 0 ]]; then
-           return 1
-        else
-           return 0
-        fi
```

Comment:
So the new two-line code returns `0` in case the AC adapter is connected, and 1 if it's not, right?

Wasn't the old code the other way around? Or maybe I'm simply confused...